gradle-plugin/sample/src/main/java/io/realm/example/sample/models/Person.java

```diff
@@ -16,10 +16,12 @@
 package io.realm.example.sample.models;
 
 import io.realm.RealmObject;
+import io.realm.annotations.Ignore;
 
 public class Person extends RealmObject {
```

Comment:
Perhaps also add a setter that mutates the data?
I assume that removing limitations on methods etc. will be another PR?